extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss

```diff
@@ -91,5 +92,142 @@
 	display: none;
 }
 
+.global-navigation-2016 {
+	.global-nav-search-input-wrapper .autocomplete-container,
+	.notifications-container .notifications-container,
```

Comment:
unfortunately yes - this is old notifications code and we are a little afraid to mangle this code.